### ISSUE TITLE
Does not work for newer NVIDIA graphics cards

### ISSUE THREAD

**Issue by shikhindahikar** [cyrusbehr/YOLOv8-TensorRT-CPP#37]
Since the OpenCV 4.8.0 with CUDA support is only available for CUDA < 12 it doesn't support newer graphics cards of 40 series and such. 40 series graphics cards can only be downgraded till 525.xx.xx driver version minimum and < 12 CUDA is available only below 520.xx.xx NVIDIA drivers.
More information in this [slack answer ](https://stackoverflow.com/a/74836282/23063523)

Is there any other way to get this code to work on newer graphics cards?

**Comment by shikhindahikar** [comment]
Oh I must have misread that slack answer. The OpenCV installation was being a huge pain and the CUDA, CUDA toolkit versions were also getting weird with each other so I thought it must be a missing feature. Sorry about that and thanks for pointing it out.
I tried installing openCV in 1660 ti machine/11.8 CUDA mostly everything went smoothly but somehow I faced lots of problems while installing for 4060 ti machine/12.2 CUDA.
